packages/flow-core/src/components/f-div/f-div.ts

```diff
@@ -270,6 +285,16 @@ export class FDiv extends FRoot {
 		if (this.height && !fixedValues.includes(this.height)) {
 			this.style.height = this.height;
 		}
+		if (this.maxWidth) {
+			this.style.maxWidth = this.maxWidth;
+		} else {
+			this.style.removeProperty("max-width");
```

Comment:
what if someone has added inline style already, this will create an issue.

can test for this markup
`<f-div style="max-width:300px"></f-div>`